### PR TITLE
[Localization] #1761 Paldean gym leaders and elite 4 dialogue Korean translation

### PR DESCRIPTION
**Pull request by returntoice**
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.

@sodaMelon @Ohmry @returntoice now participating in #1761 translating project.

## Why am I doing these changes?

Many dialogues are not translated yet.

## What did change?

All the dialogues of Paldean gym leaders and elite 4 are translated into Korean. Most parts are directly taken from the official translation.

거의 대부분 스칼렛바이올렛 대사 원문에서 그대로 가져왔으나, 일부 맥락상 수정을 한 부분도 존재합니다.
뽀삐 대사의 경우 '목수'라고 하는 게 오타라고 생각할 수도 있으나, 원문이 어린애라서 단어를 잘 모르는 설정이라 그렇습니다. 영어 원문도 'defeat'이 아니라 'defeet'이라고 쓰고 있습니다.

## How to test the changes?

It seems there won't be much to test since it's just a translation of an existing .ts file. You can check it when you meet the respective trainers.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by sodaMelon** [comment]
oh, poppy is spelling breaker.... Let us as understand children because we did samething in childhood.
(And... I think you are not used to pull&push yet. Next time, I will show you how to do that On Discord)

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -1046,24 +1046,24 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "kofu": {
     "encounter": {
-      1: "I'mma serve you a full course o' Water-type Pokémon! Don't try to eat 'em, though!"
+      1: "물포켓몬의 풀코스를! 배 터지게 먹여 주도록 하마!"
     },
     "victory": {
-      1: "Vaultin' Veluza! Yer a lively one, aren't ya! A little TOO lively, if I do say so myself!"
+      1: "우옷! 우오오옷! 이렇게 팔팔한 트레이너가 다 있다니!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:1052]: Vaultin' Veluza! ㅋㅋ 영어 문장을 끌어낼 순 없을 것같아요 ㅋㅋㅋㅋ
사투리도 섞여있었는데, 잘해주셨습니다! (포켓몬코리아에서 한국어판은 사투리를 채택하지 않았다고하네요)

File: src/locales/ko/dialogue.ts [hunk 2]
```diff
@@ -1483,14 +1483,14 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: `Fortune smiled on me this time, but… 
-                $Judging from how the match went, who knows if I will be so lucky next time.`,
+      1: `이번에는 당신이 승리를 쟁취했군요. 
+                $하지만, 시합의 흐름을 보니… 다음 승부는 또 어떻게 될지 모르겠네요.`,
     },
     "defeat": {
-      1: "That was an ace!"
+      1: "저에게 더 배우고 싶은 것이 있으시다면 또 승부하도록 하죠."
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:1493]: 원문에는 우등생이시군요<- 라고 말했지만, 한국어 정식발매 대사를 잘 활용하셨습니다.👍

File: src/locales/ko/dialogue.ts [hunk 3]
```diff
@@ -1969,60 +1969,60 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "defeat": {
-      1: "I'll call it a day."
+      1: "오늘은 저는 이만 실례하겠습니다."
     }
   },
   "ryme": {
     "encounter": {
-      1: "Come on, baby! Rattle me down to the bone!"
+      1: "나의 영혼 흔들어 봐 Come On!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:2008]: 한국어판 로컬라이징( hiphop tournament)를 잘 살려주셨어요.

**Review by sodaMelon — APPROVED** [review]
LGTM! 👍There is no problem. Excellent

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**